ai/mcp-server.md

```diff
@@ -0,0 +1,200 @@
+   - mcp__tiger__tiger_service_update_password - Update service passwords
+
+   Is there something specific you'd like to do with the TigerData MCP server?
+   ```
+
+1. **Ask a basic question about your $SERVICE_SHORTs**
+   ```shell
+   > can you list my active services please
+   ```
+   You see something like:
+   ```shell
+   ⏺ You have 3 active services, all with the same name "my-first-service":
+
+   1. tgrservice1 - READY - TimescaleDB - us-east-1 - 1 CPU/4GB - Created: 2025-09-30
+   2. tgrservice1 - READY - TimescaleDB - us-east-1 - 1 CPU/4GB - Created: 2025-09-30
+   3. tgrservice1 - READY - TimescaleDB - us-east-1 - 1 CPU/4GB - Created: 2025-09-30
+   ```
+
+1. **Manage your $SERVICE_SHORTs without having to learn how to**
+
+   For example:
+   ```shell
+   Can you create a new ai service called bob with a read replica for me please
+   ```
+   You see something like:
+   ```shell
+    tiger - Create Database Service (MCP)(name: "bob", addons: ["ai"], replicas: 1, wait: false)
+   ...
+   Do you want to proceed?
+   ❯ 1. Yes 
+   ```
+
+1. **Find best practice for things you need to do**
+
+   For example:
+   ```shell
+   >  I need to migrate a database with 25 million rows of data to Tiger, 
+   please tell me what service optimizations i should do before migration. 
+   
+   ```
+   
+   You see something like:
+
+   ```shell
+   ⏺ For migrating a database with 25 million rows to Tiger, here are the key service optimizations to do before migration:
+
+   Pre-Migration Service Optimizations
+
+   1. Right-size your compute resources:
+   - For 25M rows, start with at least 2 CPU/8GB or 4 CPU/16GB
+   - You can scale up during migration if needed, then scale down after
+
+   2. Enable the appropriate add-ons:
+   - Use time-series addon if you're working with time-series data
+   - Use ai addon if you need vector/AI capabilities
+
+   3. Choose optimal region:
+   - Select the region closest to your current database to minimize network transfer time
+   - This significantly reduces migration duration
+
+   4. Consider enhanced storage (Enterprise only):
+   - If you expect high write throughput during migration, enhanced storage provides better IOPS (up to 32,000)
+
+   5. Skip replicas initially:
+   - Start with 0 replicas during migration
+   - Add replicas after migration completes to avoid duplicate data copying
+
+   6. Plan for storage space:
+   - Estimate your uncompressed data size
+   - Allocate sufficient high-performance storage (scales automatically but good to plan)
+   - You can enable tiered storage later for older data
+
+   During Migration Best Practices
+
+   Migration method selection:
+   - 25M rows: Use schema-then-data migration (migrate schema first, then copy data in chunks)
+     - Allows restart if a copy operation fails
+     - Better for databases >10GB
+
+   Database configuration to set AFTER migration:
+   - Set up compression policies (once data is loaded)
```

Comment:
Are bullets here and in the Connection strings section supposed to be on different levels?